plugin/trino-opa/src/main/java/io/trino/plugin/opa/OpaAccessControl.java

```diff
@@ -709,6 +714,23 @@ public void checkCanDropFunction(SystemSecurityContext systemSecurityContext, Ca
                 OpaQueryInputResource.builder().function(TrinoFunction.fromTrinoFunction(functionName)).build());
     }
 
+    @Override
+    public List<ViewExpression> getRowFilters(SystemSecurityContext context, CatalogSchemaTableName tableName)
```

Comment:
Regarding `getRowFilters` and `getColumnMask`: Is my understanding correct that this is currently only implemented by having an OPA call per table (for row filter) / column (for column mask)?
Do you have any plans of implementing a batch variant as well?

Would you even go one step further to assume users using these two functionalities are advanced enough to justify only offering a batched API for code complexity reasons?

Reply:
Pinged you on Slack to discuss a bit more, will update this PR comment once I understand the options available a bit better